site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -295,6 +297,13 @@ For most use cases the controller would have required a restart anyway (eg: cont
 You can use our script located at {{< ghlink href="install/helm/agones/certs/cert.sh" >}}cert.sh{{< /ghlink >}} to generate them.
 {{< /alert >}}
 
+## Fixed Allocator Load Balancer IP
+
+If you have reserved a static IP for your allocator, in the case that the Agones allocator is installed as a `LoadBalancer` service, you can pass the static IP you have reserved to Agones with the `agones.allocator.http.loadBalancerIP`. Assuming your Kubernetes provider supports this setting a `LoadBalancer` service's IP, this will do two things:
```

Comment:
Let's try to reword this to make it even clearer. One suggestion:

In order to reuse the existing load-balancer IP on upgrade or to install the agones-allocator service using a reserved IP, a user can specify the load-balancer IP as a helm configuration value for `agones.allocator.http.loadBalancerIP`. By setting the `loadBalancerIP` value:

1. The load-balancer is created with the specified IP, if supported by the cloud provider.
2. A self-signed server TLS certificate is generated for the IP, used by the `agones-allocator` service.

Reply:
Reworded it as you suggested!